### ISSUE TITLE
Golems won't harvest or harvest very very very slowly

### ISSUE THREAD

**Issue by ncottrill** [t2pellet/strawgolem#94]
**Game Info**
Game Version :  1.18.2
Mod Version : Strawgolem-forge-1.18.2-2.0.0b1
Mod Loader : Curseforge

**Describe the bug**
Strawgolem doesn't do any harvesting. 

**To Reproduce**

1. Create a strawgolem. 
2. He finds a wheat and harvests it. 
3. Then, the strawgolem just ambles off and wanders around and does no work. 
4. I feed it an apple, then a piece wheat. It talks but does not work. After picking up an initial plant and harvesting it, they just hang out and maybe pick one piece of food per Minecraft day.

**Screenshots**
The top image is them pouting because they are hungry.
I fed them each 5 apples and enough wheat to get all their hearts to full capacity.
They stand up ... but don't do anything. No work.

![image](https://user-images.githubusercontent.com/14022160/177709996-db4b2d2d-a719-444a-a476-ae83b5baf000.png)

**Additional context**

**Comment by Clodocard** [comment]
I have the same issue, but I also noticed they try to move, but can't. They also make struggle noises, as if they are holding something very heavy.

**Comment by ncottrill** [comment]
I have discovered that only 1 chest can be in the area around them. Just 1 single chest. Then they will work ... some. In the configuration, because we were having some trouble with them, I set them to infinite life and then hungry every 14 in-game days. Each golem then eats almost an entire stack of apples! That hardly makes these little guys (cute as they are) worth having. 64 apples for 14 days work where they might pick 5-8 items in a day ... 

The important addition here is that 1 single chest will get them to function somewhat.
